library/qs_source/src/std/convert.qs

```diff
@@ -264,5 +264,37 @@ namespace Microsoft.Quantum.Convert {
         );
     }
 
-    export IntAsDouble, IntAsBigInt, ResultAsBool, BoolAsResult, BoolArrayAsInt, IntAsBoolArray, BoolArrayAsBigInt, BigIntAsBoolArray, ResultArrayAsInt, ResultArrayAsBoolArray, BoolArrayAsResultArray, ComplexAsComplexPolar, ComplexPolarAsComplex;
+    /// # Summary
+    /// Converts a given double-precision floating-point number to a string representation with desired precision, rounding if required.
+    ///
+    /// # Input
+    /// ## input
+    /// Double to be converted.
+    /// ## precision
+    /// Non-negative number of digits after the decimal point.
+    /// # Example
```

Comment:
This is not a formatter issue, but I would add an empty line here for readability, like the other operations in this file do
```suggestion
    ///
    /// # Example
```